users_app/models.py

```diff
@@ -17,9 +17,8 @@ def create_user(self, email, password=None, **extra_fields):
         if not email:
             raise ValueError(_('The Email must be set'))
         email = self.normalize_email(email)
-        if 'preferred_language' in extra_fields:
-            if extra_fields['preferred_language'] not in LANGUAGES:
-                extra_fields['preferred_language'] = 'en'
+        if extra_fields['preferred_language'] not in LANGUAGES:
```

Comment:
You can make that change but use get so if it does not have the key it will be None by default
if extra_fields.get('preferred_language') no in LANGUAGES

Reply:
okas